File: Cplusplus/URI/01-iniciantes/02-Nivel/1012.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;
double ReceiveSidesMeasurements(double sizeMensuraments[3]);
double CalculateTriagleArea(double sizeMensuraments[3]);
double CalculateCircleArea(double sizeMensuraments[3]);
double CalculateTrapeziumArea(double sizeMensuraments[3]);
double CalculateSquareArea(double sizeMensuraments[3]);
double CalculateRectangleArea(double sizeMensuraments[3]);

int main()
{
    double sizeMensuraments[3];
    ReceiveSidesMeasurements(sizeMensuraments);
    CalculateTriagleArea(sizeMensuraments);
    CalculateCircleArea( sizeMensuraments);
    CalculateTrapeziumArea( sizeMensuraments);
    CalculateSquareArea( sizeMensuraments);
    CalculateRectangleArea( sizeMensuraments);    
    return 0;
}
double ReceiveSidesMeasurements(double sizeMensuraments[3])
{
    double sideA, sideB, sideC;
    cout << "Digite os lados do A, B e C da figura." << endl;
    cin >> sideA >> sideB >> sideC >> fixed >> setprecision(2);
    sizeMensuraments[0] = sideA;
    sizeMensuraments[1] = sideB;
    sizeMensuraments[2] = sideC;   
    return sizeMensuraments[3];
}
double CalculateTriagleArea(double sizeMensuraments[3])
{
    double result = sizeMensuraments[0] * (sizeMensuraments[2] / 2);
    cout << "TRIANGULO: " << fixed << setprecision(3) << result << endl;
}
double CalculateCircleArea(double sizeMensuraments[3])
{
    double pi = 3.14159 , result;
    result = pi * pow(sizeMensuraments[2],2);
    cout << "CIRCULO: " << fixed << setprecision(3) << result << endl;
}
double CalculateTrapeziumArea(double sizeMensuraments[3])
{
    double result = ((sizeMensuraments[0] + sizeMensuraments[1]) * sizeMensuraments[2]) / 2;
    cout << "TRAPEZIO: " << fixed << setprecision(3) << result << endl;
}
double CalculateSquareArea(double sizeMensuraments[3])
{
    double result = pow(sizeMensuraments[1],2);
    cout << "QUADRADO: " << fixed << setprecision(3) << result << endl;
}
double CalculateRectangleArea(double sizeMensuraments[3])
{
    double result = sizeMensuraments[0] * sizeMensuraments[1];
    cout << "RETANGULO: " << fixed << setprecision(3) << result << endl;
}

File: Javascript/URI/01-Iniciantes/1000-1020/1012.js
sidesReceiveData = ReceiveData();

console.log("TRIANGULO: " + CalculateTriagleArea(sidesReceiveData).toFixed(3));
console.log("CIRCULO: " + CalculateCircleArea(sidesReceiveData).toFixed(3));
console.log("TRAPEZIO: " + CalculateTrapeziumArea(sidesReceiveData).toFixed(3));
console.log("QUADRADO: " + CalculateSquareArea(sidesReceiveData).toFixed(3));
console.log("RETANGULO: " + CalculateRectangleArea(sidesReceiveData).toFixed(3));

function ReceiveData()
{
    let sidesReceiveData = [];       
    sidesReceiveData = prompt("Digite o valor do lado A, B, C com espaços!").split(" ");
    for (let i = 0; i < sidesReceiveData.length; i++) {
        sidesReceiveData[i] = parseFloat(sidesReceiveData[i]);        
    }        
    return sidesReceiveData;
}
function CalculateTriagleArea(sidesReceiveData)
{
     return  area = sidesReceiveData[0] * sidesReceiveData[2] / 2;
}
function CalculateCircleArea(sidesReceiveData)
{
    const pi = 3.14159;
    return  area = pi * Math.pow(sidesReceiveData[2] , 2);
}
function CalculateTrapeziumArea(sidesReceiveData)
{    
    return area = ((sidesReceiveData[0] + sidesReceiveData[1]) * sidesReceiveData[2])/ 2
}
function CalculateSquareArea(sidesReceiveData)
{
    return area = Math.pow(sidesReceiveData[1], 2);
}
function CalculateRectangleArea(sidesReceiveData)
{
    return area = sidesReceiveData[0] * sidesReceiveData[1];
}